### DISCUSSION TITLE
Use SpringValue outside of animated factory

### DISCUSSION THREAD

**Discussion by cristobalbahe** [pmndrs/react-spring#1709] [Support]
Hello everyone!

I am breaking my head trying to understand how could I use the spring values resulting from useTransition (with an array input) in a custom component that is not any of the current targets of react-spring (web, three, konva...) and does not have an animated factory; for example rectangles in a plain canvas (no framework).  

What I usually do with useSpring is update things with the onChange function and refs, but with this case there are up to 100 items so if I could listen to the changes of the props inside of each component I could attain what I am trying.

**Comment by joshuaellis** [comment]
I'd argue this is sort of OOS of the library. Although have you tried just using the `SpringValue` class?
